packages/tooltips/src/styled/StyledTooltip.ts

```diff
@@ -73,9 +73,15 @@ const sizeStyles = ({
   }
 
   let arrowSize;
+  let arrowShift;
 
   if (hasArrow) {
-    if (size === 'small' || size === 'medium') {
+    if (size === 'small') {
+      arrowSize = margin;
+      if (['left-start', 'left-end', 'right-start', 'right-end'].includes(placement)) {
+        arrowShift = '-4px';
+      }
```

Comment:
```suggestion
      if (['left-start', 'left-end', 'right-start', 'right-end'].includes(placement)) {
        arrowShift = '-4px';
      } else {
        arrowShift = margin;
      }
```

[nit] prefer clarifying assignment